app/components/fullNode/FullNodeJointStyles.js

```diff
@@ -0,0 +1,204 @@
+// @flow
+import styled, { css } from 'styled-components';
+import { smColors } from '/vars';
+
+const Actionable = css`
+  &:hover {
+    opacity: 0.8;
+  }
+  &:active {
+    opacity: 0.6;
+  }
+`;
+
+export const PageWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  height: 100%;
+`;
+
+export const LeftPane = styled.div`
+  text-align: left;
+  flex: 2;
+  padding: 24px 0;
+  margin-right: 32px;
+`;
+
+export const RightPane = styled.div`
+  flex: 1;
+  background-color: white;
+  padding: 30px;
+  min-height: 480px;
+  max-height: 75%;
+  border: 1px solid ${smColors.borderGray};
+`;
+
+export const RightPaneInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+`;
+
+export const LeftPaneInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  height: 100%;
+`;
+
+export const RightHeaderWrapper = styled.div`
```

Comment:
any particular reason for having wrapper and then nesting inside RightHeaderText?

Reply:
nope.